jwala-integration-test/src/test/postman/jwala-collection.postman_collection.json

```diff
@@ -2202,7 +2202,7 @@
 					],
 					"request": {
 						"url": {
-							"raw": "{{url}}/v1.0/groups/{{HealthCheckGroup}}/apps/conf/{{HealthCheckWar}}/HEALTH-CHECK-4.0?hostName={{host}}",
+							"raw": "{{url}}/v1.0/groups/{{HealthCheckGroup}}/apps/conf/healthcheck-webapp-1.0.3.war/{{HealthCheckApp}}?hostName={{host}}",
```

Comment:
We should keep the name and version of the health check war as variable so we only have to change it in one place in the future.

I'm confused because you changed the group name into a variable but then changed the existing variable for the war into a hard coded value. Seems inconsistent.